rocketpy/rocket/rocket.py

```diff
@@ -2,20 +2,20 @@
 
 import numpy as np
 
-from ..mathutils.function import Function
-from ..motors.motor import EmptyMotor
-from ..plots.rocket_plots import _RocketPlots
-from ..prints.rocket_prints import _RocketPrints
-from .aero_surface import (
+from rocketpy.mathutils.function import Function
+from rocketpy.motors.motor import EmptyMotor
+from rocketpy.plots.rocket_plots import _RocketPlots
+from rocketpy.prints.rocket_prints import _RocketPrints
+from rocketpy.rocket.aero_surface import (
```

Comment:
Any particular reason for the conversion of the relative imports to absolute?

Reply:
> Any particular reason for the conversion of the relative imports to absolute?

Just to avoid circular import error, it was happening with some modules here, unfortunately. 
I honestly couldn't find where exactly the problem were, but using absolute import solved everything quickly.